radicle-cli/src/git.rs

```diff
@@ -44,6 +45,12 @@ impl Rev {
         let object = repo.revparse_single(self.as_str())?;
         Ok(ObjectId::from(object.id()))
     }
+
+    /// Resolve the revision to an [`EntryId`].
+    pub fn resolve_entry(&self, repo: &git2::Repository) -> Result<EntryId, git2::Error> {
+        let object = repo.revparse_single(self.as_str())?;
+        Ok(EntryId::from(object.id()))
+    }
 }
```

Comment:
IT should be possible to make `resolve` generic and not need two functions here. If we return `Result<T, _>` where `T: From<git2::Oid>`